data/bus-times/express.yaml

```diff
@@ -1,27 +1,26 @@
 line: Express Bus
 schedules:
   - days: [Mo, Tu, We, Th, Fr, Sa]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
     stops:
-        [St. Olaf,  Carleton,  Food Co-op, Cub/Target, El Tequila, Food Co-op, Carleton,  St. Olaf]
+        [St. Olaf,  Carleton,  Food Co-op, Papa Murphy's, Cub/Target, Arby's, Jesse James Lanes, El Tequila, Food Co-op, Carleton, St. Olaf]
```

Comment:
If you could reformat this into the one-stop-per-line format (Which is what the other two busses do) that'd be great.

Reply:
351f29a should resolve this!